apps/chromium-browser/docker-compose.yml

```
@@ -0,0 +1,16 @@
+services:
+  chromium:
+    image: lscr.io/linuxserver/chromium:stable
+    container_name: chromium
+    environment:
+      - PUID=1000
+      - PGID=1000
+      - TZ=Etc/UTC
+      - CHROME_CLI=https://www.linuxserver.io/ #optional
+    volumes:
+      - /path/to/config:/config
+    ports:
+      - 3000:3000
+      - 3001:3001
+    shm_size: "1gb"
+    restart: unless-stopped
```

We should change this to `on-failure`
```suggestion
    restart: on-failure
```